src/discussions/common/AlertBanner.test.jsx

```diff
@@ -0,0 +1,66 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import { Factory } from 'rosie';
+
+import { camelCaseObject, initializeMockApp, snakeCaseObject } from '@edx/frontend-platform';
+import { AppProvider } from '@edx/frontend-platform/react';
+
+import { ThreadType } from '../../data/constants';
+import AlertBanner from './AlertBanner';
+
+import '../comments/data/__factories__';
+
+let store;
+
+function buildTestContent(buildParams) {
+  const buildParamsSnakeCase = snakeCaseObject(buildParams);
+  return [
+    Factory.build('comment', { ...buildParamsSnakeCase }, null),
+  ].map(content => camelCaseObject(content));
+}
+
+function renderComponent(
+  content,
+) {
+  render(
+    <IntlProvider locale="en">
+      <AppProvider store={store}>
+        <AlertBanner
+          content={content}
+          postType={ThreadType.QUESTION}
+        />
```

Comment:
we are looking for word TA and staff in the alert banner anywhere can we be more specific or are we sure that Staff and TA will only be there only once in a single span? because this test will pass even if it find staff and TA anywhere in the alert banner.

Reply:
There cannot be these words unless someone includes these in their name. Yes, the test will pass if there’s such a case. Updated it.